ui/src/app/pages/login/cookie-policy.tsx

```diff
@@ -0,0 +1,161 @@
+             src={SIGNED_OUT_HEADER_IMAGE}/>
+      </div>
+      <FadeBox style={{margin: 'auto', marginTop: '1rem', width: '95.7%', color: colors.primary}}>
+        <Header style={styles.header}><AoUTitle/> Cookie Policy</Header>
+        <div style={styles.textSection}>
+          The <AoUTitle/> platform uses cookies to help provide you with the best experience we can.
+          Cookies are small text files that are placed on your computer or mobile phone when you browse websites.
+        </div>
+        <div style={styles.textSection}>
+          Our cookies help us to:
+          <ul>
+            <li>Make our platform function as you would expect</li>
+            <li>Improve the speed and security of the site</li>
+            <li>Continuously improve our platform for you</li>
+          </ul>
+        </div>
+        <div style={styles.textSection}>
+          We do not use cookies to:
+          <ul>
+            <li>Collect any personally identifiable information (without your express permission)</li>
+            <li>Collect any sensitive information (without your express permission)</li>
+            <li>Pass personally identifiable data to third parties</li>
+            <li>Pay sales commissions</li>
+          </ul>
+        </div>
+        <div style={styles.textSection}>
+          You can learn more about all the cookies we use below.
+        </div>
+        <SmallHeader style={styles.smallHeader}>Granting us permission to use cookies</SmallHeader>
+        <div style={styles.textSection}>If the settings on your software that you are using to view the <AoUTitle/> are
+          adjusted to accept cookies, we take this, and your continued use of our platform, to mean that you accept
+          this. Should you wish to not accept cookies, you can learn how to do this below.</div>
+        <SmallHeader style={styles.smallHeader}>Anonymous visitor statistics cookies</SmallHeader>
+        <div style={styles.textSection}>
+          We use cookies to compile visitor statistics including the number of visitors to our platform, technology used
+          to visit our platform (e.g. operating systems, helping to identify specific problems), time spent on the
+          platform, popular pages, etc. This helps us to continuously improve our platform. Analytics programs also tell
+          us, anonymously, how visitors found our platform (e.g. through a search engine) and whether they have been
+          here before, helping us to invest in the development of our services instead of marketing.
+        </div>
+        <div style={styles.textSection}>
+          There will also be cookies on our platform placed by third parties whose services we use to deliver our
+          services. To control third-party cookies, you can adjust your browser settings, as below.
+        </div>
+        <SmallHeader style={styles.smallHeader}>Cookies we use</SmallHeader>
+        <FlexRow>
```

Comment:
Ideally this would be a table, but with some of the styling I think it makes more sense to do it this way. If you have any other ideas, I would love them.